### DISCUSSION TITLE
Changing weak passwords

### DISCUSSION THREAD

**Discussion by hacker0ni** [Security-Onion-Solutions/securityonion#5812] [Unsupported Versions] [answered]
It looks like some secrets that are used for ElasticSearch authentication and MySQL only use ~20 character passwords. Is there a way to edit these without breaking functionality? Is there a limit on password length like on SOC password (limited to 72 characters)? Is there any documentation that we can refer to see where we can find all the credentials? Thanks in advance!

**Comment by weslambert** [comment]
We don't currently provide a way to easily manage each of these after the fact. This may be something we adjust in the future, but these are primarily generated with unique values and are passed to certain commands at some point during setup and potentially otherwise to facilitate the creation and usage of the password for the downstream application.

**Comment by hacker0ni** [comment]
Please see #6043 for changing said credentials.

**Comment by dougburks** [comment, marked as answer]
I've updated the documentation:
https://docs.securityonion.net/en/2.3/so-elastic-auth.html#service-accounts
